core/modules/node/node.module

```diff
@@ -75,6 +75,12 @@ define('NODE_ACCESS_DENY', 'deny');
  */
 define('NODE_ACCESS_IGNORE', NULL);
 
+/**
+ * Return status for saving which involved an update to an existing node which
+ * was not the approved revision.
+ */
+define('SAVED_REVISION', 4);
```

Comment:
Not sure about this being in Node module, but if we don't return a new value from `node_save()`, we don't need this new constant in the first place.